### ISSUE TITLE
Expo 48 - expo router@1.7.7 - npx expo start - Doesn't work, shows configuration error.

### ISSUE THREAD

**Issue by pixelkoduje** [expo/router#690]
### Which package manager are you using? (Yarn is recommended)

None

### Summary

I have expo 48 - and I want to leave it (I don't want to update the library to v2 - you said v1 is stable)
So what steps should be taken to make it work at expo 48? I installed expo-router@1.7.7

He gets such an error
`
ValidationError: Invalid options object. expo-router config plugin has been initialized using an options object that does not match the API schema.
 - options should be an object:
   object { origin, headOrigin?, unstable_src?, asyncRoutes? }
`

### Minimal reproducible example

up

**Comment by pixelkoduje** [comment]
When I install expo ^49.0.0-alpha.10 and expo-router v2 then I get this error. I just want to turn on expo-router and that's it.

`/bin/bash: C:\Users\KULTI\Desktop\PROJECTS\deliveroo\node_modules\.bin\..\expo\bin\cli: No such file or directory`

**Comment by AyrDS** [comment]
Hi @pixelkoduje ! I had the same error and I solved it with these lines of code in the app.json file

```
"plugins": [
      [
        "expo-router",
        {
          "origin": "https://acme.com",
          "asyncRoutes": "development"
        }
      ]
    ]
```

I hope you find it helpful

**Comment by sleeper** [comment]
> Hi @pixelkoduje ! I had the same error and I solved it with these lines of code in the app.json file
> 
> ```
> "plugins": [
>       [
>         "expo-router",
>         {
>           "origin": "https://acme.com",
>           "asyncRoutes": "development"
>         }
>       ]
>     ]
> ```
> 
> I hope you find it helpful

Great, fixed it on my side as well! Thanks!

**Comment by EvanBacon** [comment]
Fixed in 2.0.0-rc.9
